### DISCUSSION TITLE
Enable Click to Autofill for Searched Items in Browser Extensions

### DISCUSSION THREAD

**Discussion by mochasweet** [dani-garcia/vaultwarden#3230] [Ideas]
For my case, I have 20 gmails, so it's faster for me to just lookup the username instead, however, I have to view the item in order to press the autofill button, are there faster ways without viewing the item itself? Thanks

**Comment by BlackDex** [comment]
I suggest to create a feature request here: https://community.bitwarden.com/c/feature-requests/5/none

Vaultwarden is just the server backend and does nothing it self with the clients.

**Reply by mochasweet** [reply]
noted, thx
